examples/systemd/otelcol-sumo.service

```diff
@@ -2,7 +2,7 @@
 Description=Sumo Logic Distribution for OpenTelemetry Collector
 
 [Service]
-ExecStart=/usr/local/bin/otelcol-sumo --config /etc/otelcol-sumo/config.yaml
+ExecStart=/usr/local/bin/otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml
```

Comment:
I believe this should be:
```
/usr/local/bin/otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
```

So that both default `sumologic.yaml` config and globs are applied.

Reply:
glob is not working for released versions afaik

Reply:
```suggestion
ExecStart=/usr/local/bin/otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
```